src/panels/lovelace/cards/hui-gauge-card.ts

```diff
@@ -121,33 +116,40 @@ class HuiGaugeCard extends LitElement implements LovelaceCard {
       `;
     }
 
+    const sliderBarColor = this._computeSeverity(state);
+
     return html`
       <ha-card
-        @click="${this._handleClick}"
+        @click=${this._handleClick}
         tabindex="0"
         style=${styleMap({
-          "--base-unit": this._baseUnit,
+          "--round-slider-bar-color": sliderBarColor,
+          "--font-size":
+            this.clientWidth > 200
```

Comment:
Do we need a `resizeObserver` for this?

Reply:
Might not be a bad Idea. I didn't want to add more than needed. But at this point resize Observer is used everywehre